CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# 0.35.0
+
+- Propose-downstream: log when a PR already exists downstream (#1322).
+- `packit init` to set spec file path in the config if it's not defined (#1313).
+- Don't use `str.removeprefix()` since it was introduced in Python 3.9 and we still support older versions of Python 3 (#1318).
```

Comment:
implementation detail

Reply:
not really, someone could potentially run into this while using dev version of packit on python <3.9

I can drop it since we never released packit with that change

Reply:
I dropped it